README.md

```diff
@@ -92,6 +109,31 @@ php bin/console doctrine:migrations:migrate --no-interaction
 
 Want more? See the [official Symfony 4.3 documentation](https://symfony.com/doc/4.3/deployment.html) section about deployment.
 
+### Jobs
+
+#### Consumer
+
+In order to have jobs processed the queue consumer has to be running. You probably want something to watch that the process is running all the time, and take an action if it doesn't. You could use [Supervisor](http://supervisor.org) as this something with the following settings added:
+
+```ini
+[datatidy:consumer]
+process_name=%(program_name)s_%(process_num)02d
+command=/usr/bin/env php path/to/datatidy/bin/console consume async
+autostart=true
+autorestart=true
+numprocs=1
+redirect_stderr=true
+stdout_logfile=path/to/output.file
+```
+
+#### Producer
+
+You'll need to run the producer every minute to create jobs the consumer can process. You could for example use cron with the following settings to run the producer every minute: 
+
+```crontab
+* * * * * /usr/bin/env php path/to/datatidy/bin/console datatidy:data-flow:produce-jobs > path/to/output.file
```

Comment:
Does `datatidy:data-flow:produce-jobs` actually generate content that must be stored in a file (`output.file`) or are we just catching any error output in the file?

Reply:
Done.

Reply:
It doesn't produce actual output but it is for catching error output.